drv/lpc55-swd/src/main.rs

```diff
@@ -675,7 +675,7 @@ impl NotificationHandler for ServerImpl {
             sys_irq_control(notifications::JTAG_DETECT_IRQ_MASK, true);
         }
 
-        if (bits & notifications::TIMER_MASK) != 0 {
+        if bits.has_timer_fired(notifications::TIMER_MASK) {
```

Comment:
This is technically a functionality change, since it looks like I missed this code when adding the timer checks. But I think the result is correct.

Reply:
IIRC you intentionally skipped it because I was also messing with stuff in https://github.com/oxidecomputer/hubris/pull/2228 😄